File: Kata Training/8 Kyu/10-returnDay.py
# 1. Barbar
# def whatday(num):
#    if(num == 1):
#       return "Sunday"
#    elif(num == 2):
#       return "Monday"
#    elif(num == 3):
#       return "Tuesday"
#    elif(num == 4):
#       return "Wednesday"
#    elif(num == 5):
#       return "Thursday"
#    elif(num == 6):
#       return "Friday"
#    elif(num == 7):
#       return "Saturday"
#    else:
#       return "Wrong, please enter a number between 1 and 7"

# 2
def whatday(num):
   days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
   if 1 <= num <= 7:
      return days[num - 1]
   else:
      return "Wrong, please enter a number between 1 and 7"

File: CODEWARS/Kata Training/8 Kyu/10-returnDay.js
// 1. Barbar
function whatday(num) { 
   return num == 1 ? "Sunday" : 
   num == 2 ? "Monday" :
   num == 3 ? "Tuesday" :
   num == 4 ? "Wednesday" :
   num == 5 ? "Thursday" :
   num == 6 ? "Friday" :
   num == 7 ? "Saturday" :
   "Wrong, please enter a number between 1 and 7"
}

// 2.
function whatday(num) { 
   const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
   return (1 <= num && num  <= 7) ? days[num - 1] : "Wrong, please enter a number between 1 and 7"
}